go/private/BUILD.sdk.bazel

```diff
@@ -69,3 +69,10 @@ filegroup(
         "pkg/**",
     ]),
 )
+
+filegroup(
```

Comment:
Left a comment here earlier, but GitHub seems to have swallowed it.

Instead of a `filegroup`, I think this should be:

```
exports_files(
    ["lib/time/zoneinfo.zip"],
    visibility = ["//visibility:public"],
)
```